atsign-foundation/atmosphere_pro
# feat: #211 UI Redesign - Desktop - Settings

**- What I did**
Implemented the Settings screen for desktop, closes #211 

**- How I did it**
I followed the figma design given and implemented it in flutter.

**- How to verify it**
Run the desktop app and from the left navigator, scroll down to the bottom and select settings. You'll see the settings page implemented with a little navigation bar changes as well.
